docs/components/buttons/android.md

```diff
@@ -94,14 +123,30 @@ WarpButton(
 )
 ```
 
-### Legacy suport
+### Icon
+
+An optional icon can be displayed at the start of the button text. Always remember to include a content description string for accessibility :) 
```

Comment:
Just wondering, with icon description and button text next to each other, I suppose the screen reader will read up both the icon description and button text with one go? 
Have you heard with the a11y team what would be the preferred approach in such case? Should the icon description include a comma to separate it from the text?

I'm also asking because I found an [article on accessible icon buttons](https://www.sarasoueidan.com/blog/accessible-icon-buttons/) that suggests setting aria-hidden on the icon when it's rendered next to the button text, provided the icon gives no more context to what the button does. But that might not be what the a11y would suggest 😅

Reply:
Yeah good point! I was thinking that it would complement each other in a way like - "Clock icon" "Set Time" 
But maybe you are right in this case perhaps the icon doesnt need any extra description to avoid confusion. Ill check with the a11y team

Reply:
a11y team suggest that both the icon and the text should be read up.